frontend/packages/db-structure/src/parser/prisma/index.test.ts

```diff
@@ -12,11 +13,11 @@ describe(_processor, () => {
           columns: {
             id: aColumn({
               name: 'id',
-              type: 'serial',
+              type: 'bigserial',
               default: 'autoincrement()',
```

Comment:
Q: I want to know the reason for splitting the userTable function into separate files. Am I understanding this `default` difference correctly?

Reply:
You're absolutely right. The biggest reason is that Prisma automatically sets the default to ``autoincrement()``.
And this default value is necessary to differentiate between ``bigint`` and ``bigserial``.

![ss 2773](https://github.com/user-attachments/assets/a1c5780a-84d2-4f32-833c-226f409c6afd)


https://www.prisma.io/docs/orm/reference/prisma-schema-reference#postgresql-3

Reply:
OK, I understand!